sonar-db/src/main/java/org/sonar/db/issue/IssueFilterFavouriteDao.java

```diff
@@ -51,16 +52,24 @@ public IssueFilterFavouriteDto selectById(long id) {
     }
   }
 
+  /**
+   * @deprecated since 5.2 use {@link #insert(DbSession, IssueFilterFavouriteDto)}
+   */
+  @Deprecated
   public void insert(IssueFilterFavouriteDto filter) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)